main.cpp

```diff
@@ -105,7 +98,9 @@ void variadicHelper(T&& first, Args&& ... args)
 
 int main()
 {
-    variadicHelper( 3, std::string("burgers"), 2.5, Point{3.f, 0.14f} );
+    variadicHelper( 3, std::string("burgers"), 2.5, Point{3.f, 0.14f} );
+
+    variadicHelper();

```

Comment:
Why are you calling this?  the zero-argument function is empty and does nothing.
- [ ] revise

Reply:
Was checking to see if any errors would be thrown. I will remove it.